src/Services/MailSender.php

```diff
@@ -85,6 +97,30 @@ private function getCurrentSalesChannelId(): ?string
         return $salesChannelId;
     }
 
+    private function getSourceMailId(Context $context): ?string
+    {
+        $request = $this->requestStack->getMainRequest();
+        if($request === null) {
+            return null;
+        }
+
+        $route = $request->attributes->getString('_route');
```

Comment:
getString has been introduced with `symfony/http-foundation:6.3` which is not available for <shopware:6.5.2.0

Reply:
I have changed it to use a simple get, which is enough in this case.